Automattic/wp-calypso
# Staging Sites: Prevent creating a staging site when jetpack is unhealthy

Fixes https://github.com/Automattic/dotcom-forge/issues/3536

## Proposed Changes

This diff prevents creating a staging site if Jetpack is unhealthy

## Testing Instructions



1. Apply D119844-code
2. Create a WoA site. 
3. Use phpmyadmin to delete `wp_options` table.
4. Visit `/hosting-config` page.
5. Make sure that you see the following message, and the `Add Staging Site` button is disabled:
![create_staging_site_jetpack_message](https://github.com/Automattic/wp-calypso/assets/497103/ce818eee-4562-48a9-b3c6-9b69ca87c782)



## Pre-merge Checklist
